tests/dev/CsvBulkLoaderTest.php

```diff
@@ -229,6 +229,17 @@ protected function getLineCount(&$file) {
 		return $i;
 	}
 
+	public function testLargeFileSplitIntoSmallerFiles() {
+		Config::inst()->update('CsvBulkLoader', 'lines', 10);
+
+		$loader = new CsvBulkLoader('CsvBulkLoaderTest_Player');
+		$path = $this->getCurrentAbsolutePath() . '/CsvBulkLoaderTest_LargeListOfPlayers.csv';
+
+		$results = $loader->load($path);
+
+		$this->assertEquals(100, $results->Count());
```

Comment:
This will cause a hundred row writes for the test, which is a bit excessive in terms of keeping our tests fast. How about using 10 lines total, and chunking each 3 lines?